apps/auth0-rules/src/add-user-metadata.ts

```diff
@@ -14,7 +14,9 @@ const addUserMetadata: Rule<{ invitationCode: string }> = async (
 
   const redirect_uri = new URLSearchParams(context.request.query).get(
     'redirect_uri',
-  );
+  )
+    ? new URLSearchParams(context.request.query).get('redirect_uri')
```

Comment:
Umm... `??`? :D

Reply:
I did this initially but it compiles weirdly making the manual adjustments harder

Reply:
we need to do manual adjustments here? o.O
thought only wrapping the whole thing in a fn, and the dep version lockdown